### ISSUE TITLE
Unable to Associate Element

### ISSUE THREAD

**Issue by tyronewyatt** [InfotelGLPI/webapplications#3]
Web Applications not an Associate Element type when adding to a Ticket/Problem/Changes using plugin version 2.0.0 = > on GLPI 0.58.0 = >.
Feature last known working in version 1.9.1 with GLPI 0.84.x.
http://forum.glpi-project.org/viewtopic.php?id=46249

**Comment by tsmr** [comment]
Into profile : add it to array Associable items to a ticket ?

**Comment by tyronewyatt** [comment]
Yes associable items  to a ticket is enabled in the profiles. See my screenshots attached which confirms the option is enabled in GLPI 0.84.8 Web application 1.91 and in GLPI 0.85.2 Web application 2.0.0.
Checked for erros in glpi\files\_log but none found.

glpi 0 84 - create ticket
![glpi 0 84 - create ticket](https://cloud.githubusercontent.com/assets/13756677/11197522/1292ce42-8d12-11e5-9ca5-f786324448c9.PNG)
glpi 0 85 - create ticket
![glpi 0 85 - create ticket](https://cloud.githubusercontent.com/assets/13756677/11197518/1127a410-8d12-11e5-9ee2-34c74fc32f12.PNG)
glpi 0 84 - profiles
![glpi 0 84 - profiles](https://cloud.githubusercontent.com/assets/13756677/11197517/11260b00-8d12-11e5-84bb-a07b22df789c.PNG)
glpi 0 85 - profiles
![glpi 0 85 - profiles](https://cloud.githubusercontent.com/assets/13756677/11197519/1155ae6e-8d12-11e5-804d-6e89f06e3981.PNG)
glpi 0 84 - plugins
![glpi 0 84 - plugins](https://cloud.githubusercontent.com/assets/13756677/11197868/79c4b09c-8d14-11e5-9f8b-60fe83f30952.PNG)
glpi 0 85 - plugins
![glpi 0 85 - plugins](https://cloud.githubusercontent.com/assets/13756677/11197523/12968258-8d12-11e5-83a8-66c9dfe41698.PNG)
glpi 0 84 - web application tickets
![glpi 0 84 - web application tickets](https://cloud.githubusercontent.com/assets/13756677/11197516/10f8d554-8d12-11e5-9a5e-00db84648673.PNG)
glpi 0 85 - web application tickets
![glpi 0 85 - web application tickets](https://cloud.githubusercontent.com/assets/13756677/11197520/115748e6-8d12-11e5-8da6-408f1f42a74b.PNG)
glpi 0 84 - web application glpi
![glpi 0 84 - web application glpi](https://cloud.githubusercontent.com/assets/13756677/11197515/10f80796-8d12-11e5-9f52-253ee113344c.PNG)
glpi 0 85 - web application glpi
![glpi 0 85 - web application glpi](https://cloud.githubusercontent.com/assets/13756677/11197521/1185bcc6-8d12-11e5-96f7-21c053aee45b.PNG)

**Comment by tyronewyatt** [comment]
This issue also exists in Web applications 2.1.1 on GLPI 0.90. 

glpi 0 90 - web application tickets
![glpi 0 90 - web application tickets](https://cloud.githubusercontent.com/assets/13756677/11197983/4ef126f6-8d15-11e5-9ae7-e7170a58ca05.PNG)
glpi 0 90 - create ticket
![glpi 0 90 - create ticket](https://cloud.githubusercontent.com/assets/13756677/11197981/4ee8752e-8d15-11e5-8c31-3d11eb53cda7.PNG)
glpi 0 90 - web application glpi
![glpi 0 90 - web application glpi](https://cloud.githubusercontent.com/assets/13756677/11197985/4f1740de-8d15-11e5-8f6d-f282273fab6b.PNG)
glpi 0 90 - profiles
![glpi 0 90 - profiles](https://cloud.githubusercontent.com/assets/13756677/11197982/4ee993f0-8d15-11e5-92f9-0e1852da09b8.PNG)
glpi 0 90 - plugins
![glpi 0 90 - plugins](https://cloud.githubusercontent.com/assets/13756677/11197984/4ef12868-8d15-11e5-81ce-cc3ac4dd3080.PNG)

**Comment by tsmr** [comment]
No into Helpdesk tab on profile (Associable items to a ticket) - not into webapplications tab

**Comment by tyronewyatt** [comment]
Yes sorry this is not an issue.
Once I added Web Application as an associable item to tickets in profiles and re-applied the profile by switching profiles or logging off and back on I was able to associate Web Applications into Tickets.
Thank you for your assistance. 
I did think it was a bug but I was wrong. Now I can proceed with upgrading 4 sites with GLPI from 0.84 to 0.85. In the future I will create a forum thread on glpi-project.org  


glpi-0 90-profile-assistance-add-associable-item-to-a-ticket]
![glpi-0 90-profile-assistance-add-associable-item-to-a-ticket](https://cloud.githubusercontent.com/assets/13756677/11314577/eb660420-903a-11e5-92dd-a17475083a91.png)
glpi-0 90-profile-assistance-added-associable-item-to-a-ticket
![glpi-0 90-profile-assistance-added-associable-item-to-a-ticket](https://cloud.githubusercontent.com/assets/13756677/11314578/eb6701a4-903a-11e5-87cf-dafb101675e7.png)
glpi-0 90-ticket-added-web-application-associated-element
![glpi-0 90-ticket-added-web-application-associated-element](https://cloud.githubusercontent.com/assets/13756677/11314576/eb638952-903a-11e5-811e-6551f8e2c6bc.png)
